css/src/tokens/colors.scss

```diff
@@ -136,7 +139,9 @@ $colors: (
 		$primary-active,
 		$primary-invert,
 		$primary-box-shadow,
-		$primary-background-glow-high-contrast
+		$primary-background-glow-high-contrast,
```

Comment:
Just confirming, do we need the other -background-hover and dark-hover items in this map?

Reply:
They do need to be added at some point. I'm thinking that we can add them to each map as its colors are updated and finalized. I'm also not opposed to adding them now.

Reply:
Probably best to add them now, in my opinion. That work?

Reply:
Added